chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -764,12 +763,7 @@ impl PeerManagerActor {
                     NetworkResponses::RouteNotFound
                 }
             }
-            NetworkRequests::StateRequestHeader { shard_id, sync_hash, target } => {
-                let peer_id = match target {
-                    AccountOrPeerIdOrHash::AccountId(_) => return NetworkResponses::RouteNotFound,
-                    AccountOrPeerIdOrHash::PeerId(peer_id) => peer_id,
-                    AccountOrPeerIdOrHash::Hash(_) => return NetworkResponses::RouteNotFound,
```

Comment:
What was the point of account id and hash if it always returned RouteNotFound?

Reply:
AccountId used to work because @VanBarbascu changed the messages to be Direct instead of Routed.
Hash hasn't worked ever, no idea what it was for.

Reply:
The network changes added made the state sync messages direct so we no longer route them to `AccuntId`.